website/pages/docs/agent/config-entries/ingress-gateway.mdx

```diff
@@ -0,0 +1,134 @@
+---
+layout: docs
+page_title: 'Configuration Entry Kind: Ingress Gateway'
+sidebar_title: ingress-gateway
+description: >-
+  The ingress-gateway config entry kind allows for configuring Ingress gateways
+  with listeners that expose a set of services outside the Consul service mesh.
+---
+
+# Ingress Gateway
+
+-> **1.8.0+:** This config entry is available in Consul versions 1.8.0 and newer.
+
+  The `ingress-gateway` config entry kind allows for configuring Ingress gateways
+  with listeners that expose a set of services outside the Consul service mesh.
+  See [Ingress Gateway](/docs/connect/ingress_gateway) for more information.
+
+## Wildcard service specification
+
+  Ingress gateways can optionally target all services within a namespace by
```

Comment:
```suggestion
  Ingress gateways can optionally target all services within a Consul namespace by
```
Do you think K8s users would be confused if we didn't specify Consul? Could they think we mean k8s namespace?

Reply:
It's possible. Changed these to "Consul namespace".